### PR TITLE
AT-DOCS-2368 update readme and add examples

### PR DESCRIPTION
**Pull request by alex-thomas-aerospike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,9 +182,239 @@ The service will skip the next startup until the previous backup run is complete
...
+### Restore backup (direct restoration)
+
+#### Direct Restore Using a Specific Backup
+
+Destination field says where to restore to. It can be one of the clusters we read in 1st section, or any other Aerospike cluster.
+
+This request restores a backup from a specified path to a designated destination.
+The `no-generation` parameter allows overwriting of existing keys if set to `true`.
+
+In the `source` section, `path` is the `key` value returned as a response in the [Full Backup List](#full-backup-list) example. The `type` parameter under `source` denotes S3 storage if set to `1` and local storage if set to `0.`
```
Comment (reugn) [inline README.md:340]:
```suggestion
In the `source` section, `path` is the `key` value returned as a response in the [Full Backup List](#full-backup-list) example. The `type` parameter under `source` denotes S3 storage if set to `1` and local storage if set to `0`.
```

File: README.md [hunk 2]
```diff
@@ -177,9 +182,239 @@ The service will skip the next startup until the previous backup run is complete
...
+
+This section details how to fetch configurations for clusters, policies, and storage options. This is useful for setting up or verifying the configuration of your system.
+
+#### Get cluster configuration
+
+This endpoint returns the configurations of existing clusters, including the default cluster setup with seed nodes and credentials.
+
+Request:
+```http
+  GET {{baseUrl}}/v1/config/clusters
```
Comment (reugn) [inline README.md:204]:
```suggestion
GET {{baseUrl}}/v1/config/clusters
```

File: README.md [hunk 3]
```diff
@@ -177,9 +182,239 @@ The service will skip the next startup until the previous backup run is complete
...
+      "key": "localStorage/filterBySetAndBin/backup/source-ns8"
+    }
+  ]
+}
+```
+</details>
+
+### Restore backup (direct restoration)
+
+#### Direct Restore Using a Specific Backup
```
Comment (reugn) [inline README.md:333]: Inconsistent capitalization of titles (while lowercasing previous ones).

File: README.md [hunk 4]
```diff
@@ -177,9 +182,239 @@ The service will skip the next startup until the previous backup run is complete
...
+      "password": "psw"
+    }
+  }
+}
+```
+</details>
+
+#### Get routine configuration
+
+Retrieves the configured backup routines, including policies, source clusters, storage options, and scheduling.
```
Comment (reugn) [inline README.md:230]: This endpoint returns only the configured routines.

File: README.md [hunk 5]
```diff
@@ -177,9 +182,239 @@ The service will skip the next startup until the previous backup run is complete
...
+    "set-list": ["backupSet"],
+    "bin-list": ["backupBin"]
+  }
+}
+```
+</details>
+
+#### Get storage configuration
+
+Shows the storage configurations available, including local and MinIO (S3) storage options.
```
Comment (reugn) [inline README.md:264]: The description should be more specific. It returns all the configured storage endpoints.

**Review by jd-aero — APPROVED** [review]
+1 (I may review the whole file and submit some general suggestions)

**Review by reugn — APPROVED** [review]

**Pull request merged**